src/Microsoft.IdentityModel.Tokens/CertificateHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Helper class to load X509Certificate2 from byte array.
+    /// </summary>
+    internal class CertificateHelper
+    {
+        public static X509Certificate2 LoadX509Certificate(byte[] data)
```

Comment:
If you wanted to, you could add another overload that takes a string in base64format, then convert in the method.

Reply:
I promise I will do this in a follow up PR, I don't want to reset any approvals at this stage 🙏 🙇

Reply:
As promised: #2834